tests/linting/TerminusStandard.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<ruleset name="TerminusStandard">
+  <description>PSR2 with comments having no max line length.</description>
+  <rule ref="PSR2">
+    <exclude name="Generic.Files.LineLength" />
+  </rule>
```

Comment:
The new rule doesn't need to be referenced here. It is found automatically based on the location of this file.